python/README.rst

```diff
@@ -0,0 +1,47 @@
+Veles Python library
+====================
+
+This library enables you to operate on veles objects tree from python.
+It connects to running veles instance via network.
+
+Installing
+----------
+
+First you need to copy ``network_pb2.py`` to ``veles`` directory (this file is
+generated during Veles compilation).
+
+Then you need to run ``python setup.py install`` (or ``develop`` based on your preferences)
```

Comment:
Once again a good point. Maybe we can just write a text file with path or something?